railsconf/2020/let_juniors_fail.md

```diff
@@ -0,0 +1,36 @@
+# Give you juniors space to fail/Benefits of allowing junior developers to fail
+
+## Abstract
+
+In this talk, we will discuss the many different ways you can create an environment that promotes creativity and experimentation for junior developers. To create an environment that fosters mentorship, rapid growth, and respect, you must be willing to let your junior developers fail. Inversely, if you are a junior developer, you should seek out the type of environments that will not only allow, but encourage you to push push boundaries and learn from the resulting success or failure. This talk will focus on how to be a good mentor, and how to walk with your juniors instead of in front of them by learning to harness the unchecked optimism of your junior developers.
+
+## Details
+
+This talk is mainly aimed at mentors and junior developers. The main takeaway for mentors and senior developers is a better understanding, or gentle reminder, that they can help level up junior developers simply by getting out of their way and giving them the space to explore their ideas. For juniors, this talk will give them an insight into what a great mentor is, and what they should be looking for in a mentor.
```

Comment:
This talk is aimed at junior developers and the patient heroes that mentor them.

Reply:
Perhaps mention that you encourage them to be assertive and see recruiting top mentors as an important part of their career journey.